### ISSUE TITLE
Cannot import `multicompartment_head_project`

### ISSUE THREAD

**Issue by mshavliuk** [sampsapursiainen/zeffiro_interface#197]
## Description

I can't import the multicompartment head project from [this directory](https://github.com/sampsapursiainen/zeffiro_interface/tree/master/scripts/scripts_for_importing/multicompartment_head_project), even though it was possible recently

## Steps to reproduce

Run via import example:
```matlab
addpath scripts
zef_import_example
```
Gives me this error:
```
Check for incorrect argument data type or missing argument in call to function 'struct2cell'.

Error in zef_import_parcellation_colortable (line 39)
    parcellation_aux = {parcellation_aux{1:2}, struct2cell(parcellation_aux{3})};

Error in zef_import_segmentation (line 271)
        zef = zef_import_parcellation_colortable(zef,atlas_colortable_filename,compartment_tag,atlas_merge);

Error in zeffiro_interface (line 415)
        zef = zef_import_segmentation(zef);
```

Alternatively, when I try to import the same project manually, I get exactly the same error, although I can see that some parts of the project are successfully imported. See the screenshot below

## Expected behavior

The project gets imported.

## Screenshots

<img width="1492" alt="Screenshot 2023-05-06 at 17 56 07" src="https://user-images.githubusercontent.com/6589665/236631710-25d11a2b-e6ef-4081-9127-19a30518f0da.png">

## Context

| Name | Value |
| --- | --- |
| **Version** | 2241fdd zef_nse_sigma: fix indentations with Matlab's smart indent tool |
| **OS** | MacOS |
| **Shell** | N/A |

**Comment by mshavliuk** [comment]
This issue is not applicable to the `main_development_branch` branch and was noticed only in `192-fs2zefsh-improvements`

**Comment by SeSodesa** [comment]
It looks like the order of struct fields inside of the color table files generated by `fs2zef.sh` are different between the old `multicompartment_head_project` and the newer segmentations generated by `fs2zef.m`. Previously it was `parcellation_aux{1}` that contained the `struct`, but in the newer segmentations it is `parcellation_aux{3}` that contains it.

This is a bit strange since the Matlab function that saves the color table files, and which is called by `fs2zef.m`, is the same as before. It might also be that the segmentation in `multicompartment_head_project` was generated with an older version of FreeSurfer. With the new segmentations, the line that now causes errors works just fine, at least with s certain `.nii` file.

**Comment by SeSodesa** [comment]
@mshavliuk I added an order-agnostic way of loading the fields from the color table files generated by the function called by `fs2zef.m` to the HEAD of `192-fs2zefsh-improvements`. I tested it with a newer segmentation and the older one in

    scripts/scripts_for_importing/multicompartment_head_project

and they both seem to pass without errors. What still needs testing is the mesh generation, since it relies on the imported segmentation being correct. If it works with both newer and older segmentations, then the relevant branch can be merged to `main_development_branch`.
